ci/fireci/fireci/commands.py

```diff
@@ -26,19 +26,34 @@
     help='GRADLE_OPTS passed to the gradle invocation.')
 @ci_command('gradle')
 def gradle_command(task, gradle_opts):
-  """Runs the specified gradle commands."""
-  gradle.run(*task, gradle_opts=gradle_opts)
+    """Runs the specified gradle commands."""
+    gradle.run(*task, gradle_opts=gradle_opts)
 
 
 @ci_command()
-def smoke_tests_experimental():
-  """Builds all SDKs in release mode and then tests test-apps against them."""
-  gradle.run('publishAllToBuildDir')
-
-  cwd = os.getcwd()
-  gradle.run(
-      'connectedReleaseAndroidTest',
-      gradle_opts='-Dmaven.repo.local={}'.format(
-          os.path.join(cwd, 'build', 'm2repository')),
-      workdir=os.path.join(cwd, 'test-apps'),
-  )
+def smoke_tests_proguarded_app():
```

Comment:
both tasks look almost identical, did you consider having a single parameterized `smoke_tests` command with a `@click.option('--build-type')`?